.github/workflows/tox-test.yml

```diff
@@ -0,0 +1,103 @@
+name: Tox tests
+
+on: [push, pull_request]
+
+jobs:
+  py27:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Intsall RPM
+        run: sudo apt-get install -y rpm
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 2.7
+      - name: Install Tox
+        run: pip install tox
+      - name: Run Tox
+        run: tox -e py27
+  static:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Intsall RPM
+        run: sudo apt-get install -y rpm
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Install Tox
+        run: pip install tox
+      - name: Run Tox
+        run: tox -e static
+  py36-pidiff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Intsall RPM
+        run: sudo apt-get install -y rpm
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Install Tox
+        run: pip install tox
+      - name: Run Tox
+        run: tox -e pidiff
+  coverage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Intsall RPM
+        run: sudo apt-get install -y rpm
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Install Tox
+        run: pip install tox
+      - name: Install pytest cov
+        run: pip install pytest-cov
```

Comment:
This doesn't seem quite right, because if we already have a "cov" environment in tox.ini (implied by next command "tox -e cov"), then tox should already take care of installing pytest-cov.  So this command should be redundant... can we get rid of it?

Reply:
Codecov needs coverage library which is installed with pytest-cov. When pytest-cov is installed with tox, it gets installed in a virtual environment and coverage is not available to codecov in next step. 
If we don't install pytest-cov with tox, then it fails to run tests and generate reports, so had to include pytest-cov installation steps in both tox as well as coverage workflow.

Reply:
I see, this makes sense, but may I suggest the following then:

- this shouldn't be pytest-cov specifically because codecov-action doesn't depend on "pytest-cov", it only depends on "coverage" (i.e. not the pytest stuff) - right?
- could we please move this to after we run "tox -e cov", i.e. just before we run codecov-action?  I think that makes the intent clearer, that we're installing this just for that action.  A brief comment mentioning that would also be nice.